### DISCUSSION TITLE
error

### DISCUSSION THREAD

**Discussion by DengNingyuan** [castorini/anserini#2316] [Q&A]
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  35:08 min
[INFO] Finished at: 2023-12-23T17:30:13+08:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:3.1.0:test (default-test) on project anserini: 
[ERROR] 
[ERROR] Please refer to /home/dengny/kgi-slot-filling-main/anserini/target/surefire-reports for the individual test results.
[ERROR] Please refer to dump files (if any exist) [date].dump, [date]-jvmRun[N].dump and [date].dumpstream.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException

**Comment by lintool** [comment]
You'll need to provide more details from the error trace on what failed. What Java version are you using?

**Reply by lintool** [reply]
Not hearing any follow-up, closing.
